frame/support/src/dispatch.rs

```diff
@@ -2084,4 +2107,11 @@ mod tests {
 		assert_eq!(<Test as WeighBlock<u32>>::on_finalize(2), 10);
 		assert_eq!(<Test as WeighBlock<u32>>::on_finalize(3), 0);
 	}
+
+	#[test]
+	fn call_metadata() {
+		let metadata = Call::<TraitImpl>::aux_3().get_call_metadata();
+		let expected = CallMetadata { function_name: "aux_3".into(), pallet_name: None };
+		assert_eq!(metadata, expected);
+	}
```

Comment:
we could also do a test for the outer_call metadata, maybe in frame/support/src/lib.rs there is a test for metadata we can use impl_outer_dispatch and test it there 🤔